performance_manager/tests/test_postgres.py

```diff
@@ -30,21 +76,332 @@ def set_environment_variables() -> Iterator[None]:
                 if line.startswith("#") or line == "":
                     continue
                 key, value = line.split("=")
-                logging.info("setting %s to %s", key, value)
+                logging.debug("setting %s to %s", key, value)
                 os.environ[key] = value
 
+
+@pytest.fixture(scope="module", name="db_manager")
+def fixture_db_manager() -> DatabaseManager:
+    """
+    generate a database manager for all of our tests
+    """
+    set_env_vars()
+    db_manager = DatabaseManager()
+    return db_manager
+
+
+@pytest.fixture(autouse=True, name="s3_patch")
+def fixture_s3_patch(monkeypatch: MonkeyPatch) -> Iterator[None]:
+    """
+    insert a monkeypatch over s3 related functions. our tests use local
+    files instead of s3, so we read these files differently
+    """
+
+    def mock_get_static_parquet_paths(
+        table_type: str, feed_info_path: str
+    ) -> List[str]:
+        """
+        instead of mocking up s3 responses, just rewrite this method and
+        monkeypatch it
+        """
+        logging.debug(
+            "Mock Static Parquet Paths table_type=%s, feed_info_path=%s",
+            table_type,
+            feed_info_path,
+        )
+        table_dir = feed_info_path.replace("FEED_INFO", table_type)
+        files = [os.path.join(table_dir, f) for f in os.listdir(table_dir)]
+        logging.debug("Mock Static Parquet Paths return%s", files)
+        return files
+
+    monkeypatch.setattr(
+        "lib.l0_gtfs_static_table.get_static_parquet_paths",
+        mock_get_static_parquet_paths,
+    )
+
+    def mock__get_pyarrow_table(
+        filename: Union[str, List[str]],
+        filters: Optional[Union[Sequence[Tuple], Sequence[List[Tuple]]]] = None,
+    ) -> pyarrow.Table:
+        logging.debug(
+            "Mock Get Pyarrow Table filename=%s, filters=%s", filename, filters
+        )
+        active_fs = fs.LocalFileSystem()
+
+        if isinstance(filename, list):
+            to_load = filename
+        else:
+            to_load = [filename]
+
+        if len(to_load) == 0:
+            return pyarrow.Table.from_pydict({})
+
+        return parquet.ParquetDataset(
+            to_load, filesystem=active_fs, filters=filters
+        ).read_pandas()
+
+    monkeypatch.setattr(
+        "lib.s3_utils._get_pyarrow_table", mock__get_pyarrow_table
+    )
     yield
 
 
-def test_environ() -> None:
-    """simple test stub"""
-    # ensure that the environment has been stook up correctly.
-    assert int(os.environ.get("BOOTSTRAPPED", 0)) == 1
+def check_logs(caplog: pytest.LogCaptureFixture) -> None:
```

Comment:
fun little function to check that we are logging things correctly and that the tests aren't swallowing errors.